deployment/helm/values.yaml

```diff
@@ -132,6 +215,27 @@ debugEcho:
     targetPort: 6052
     # protocol is the service protocol of the instance service
     protocol: TCP
+  # discovery defines a set of values for a debugEcho discovery handler DaemonSet
+  discovery: 
+    # enabled defines whether discovery handler pods will be deployed in a slim Agent scenario
+    enabled: false
+    image:
+      # repository is the container reference
+      repository: ghcr.io/deislabs/akri/debug-echo-discovery
+      # tag is the container tag
+      # debugEcho.yaml will default to v(AppVersion)-dev
```

Comment:
Might default to v(AppVersion)

Reply:
Fixed docs to specify when `-dev` is added 34e97dcf0